### ISSUE TITLE
onChange with range - doesn't fire until second click

### ISSUE THREAD

**Issue by slutske22** [wojtekmaj/react-calendar#318]
With `selectRange={true}`, I noticed that the `onChange` event does not fire until the user clicks *twice*.  I guess there is some internal mechanism where when `selectRange` is true, the `value` prop is not registered in the `onChange` function until the second click?  Check out the gif:

![react-calendar-onChange-range-issue](https://user-images.githubusercontent.com/52637023/74856214-9bf24600-52f6-11ea-8351-68f27e9fa959.gif)

The gif starts with '4' already selected.  One you click '12', the onChange event fires and I have an array of the start and end values of what is currently selected.  But then when I click away from the range and click '26', nothing happens.  It is not until I click '26' *again* that the onChange event fires and I have a proper value for the new date.

I'm having to use `onChange` because it seems that `onClickDay` does not properly return an array of the start and end values of a range, even with `returnValue={'range'}`.  But I feel this is a separate issue.

**Comment by wojtekmaj** [comment]
That's intended behavior, but it's going to be improved in #331.
